CHANGELOG.md

```diff
@@ -1,7 +1,14 @@
 Releases
 ========
 
-v0.2.0 (unreleased)
+v0.3.0 (unreleased)
+------------------
+
+-   **Breaking**: Most keywords reserved by Apache Thrift are now disallowed as
```

Comment:
It's really "all" but if they add more in the future, this is no longer true. But you're right. I'll change this.